// react-simple-dropdown/lib/components/DropdownTrigger.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var DropdownTrigger = (0, _react.createClass)({
  displayName: 'DropdownTrigger',
  propTypes: {
    children: _react.PropTypes.any,
    className: _react.PropTypes.string,
    onClick: _react.PropTypes.func
  },
  getDefaultProps: function getDefaultProps() {
    return {
      className: ''
    };
  },
  render: function render() {
    var _props = this.props;
    var children = _props.children;
    var className = _props.className;
    var onClick = _props.onClick;

    var classes = 'dropdown__trigger ' + className;
    return _react2.default.createElement(
      'a',
      {
        style: this.props.style,
        className: classes,
        href: '#dropdown-trigger',
        onClick: onClick
      },
      children
    );
  }
});

exports.default = DropdownTrigger;
